backend/dataall/modules/mlstudio/db/mlstudio_repositories.py

```diff
@@ -2,30 +2,44 @@
 DAO layer that encapsulates the logic and interaction with the database for ML Studio
 Provides the API to retrieve / update / delete ml studio
 """
+import stat
+from typing import Optional
 from sqlalchemy import or_
 from sqlalchemy.sql import and_
 from sqlalchemy.orm import Query
 
+from dataall.base.utils import slugify
 from dataall.base.db import paginate
-from dataall.modules.mlstudio.db.mlstudio_models import SagemakerStudioUser
+from dataall.modules.mlstudio.db.mlstudio_models import SagemakerStudioDomain, SagemakerStudioUser
 from dataall.core.environment.services.environment_resource_manager import EnvironmentResource
+from dataall.core.environment.services.environment_service import EnvironmentService
+from dataall.base.utils.naming_convention import (
+    NamingConventionService,
+    NamingConventionPattern,
+)
 
 
 class SageMakerStudioRepository(EnvironmentResource):
     """DAO layer for ML Studio"""
     _DEFAULT_PAGE = 1
     _DEFAULT_PAGE_SIZE = 10
 
```

Comment:
Reviewed that session is passed as argument to all functions of SageMakerStudioRepository. All good!